x-pack/legacy/plugins/lens/index.ts

```diff
@@ -26,10 +26,11 @@ export const lens: LegacyPluginInitializer = kibana => {
       app: {
         title: NOT_INTERNATIONALIZED_PRODUCT_NAME,
         description: 'Explore and visualize data.',
-        main: `plugins/${PLUGIN_ID}/index`,
+        main: `plugins/${PLUGIN_ID}/redirect`,
```

Comment:
It doesn't look like we have a '/redirect' route in Lens at all. I'm assuming this is a standard Kibana app thing or something? Just wanted to double check that this was an intentional change.

Reply:
This isn't a route, it's the entrypoint of the `app/lens` "app" which doesn't exist anymore. It points to the `redirect.ts` file which does the redirect to the actual app which is loaded as part of the kibana app.

All of this is an interim measure, with the new platform we can just use the application service directly.